go.mod

```diff
@@ -26,6 +28,25 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/andybalholm/brotli v1.0.5 // indirect
+	github.com/apache/thrift v0.16.0 // indirect
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/google/flatbuffers v2.0.8+incompatible // indirect
+	github.com/klauspost/asmfmt v1.3.2 // indirect
+	github.com/klauspost/compress v1.16.0 // indirect
+	github.com/klauspost/cpuid/v2 v2.0.9 // indirect
+	github.com/minio/asm2plan9s v0.0.0-20200509001527-cdd76441f9d8 // indirect
+	github.com/minio/c2goasm v0.0.0-20190812172519-36a3d3bbc4f3 // indirect
+	github.com/zeebo/xxh3 v1.0.2 // indirect
+	golang.org/x/mod v0.8.0 // indirect
+	golang.org/x/tools v0.6.0 // indirect
+	golang.org/x/xerrors v0.0.0-20220609144429-65e65417b02f // indirect
+)
+
+// replace github.com/apache/arrow/go/v12 => github.com/cloudquery/arrow/go/v12 v12.0.0-20230403113355-afc433992be2
+replace github.com/apache/arrow/go/v12 => ../arrow/go
```

Comment:
We will need to update this before merging. @yevgenypats what changes / branch are you using to test this locally?

Reply:
Oh, seems like it's `cqmain1`

Reply:
I've updated the `cqmain` branch and update the go.mod reference here

Reply:
Seems it's missing `array.DiffString` somehow. maybe we need to rebase `cqmain` on top of `main`

Reply:
I think I've resolved that error now (maybe make sure you've pulled the latest), but I'm seeing some new errors that we're also getting in CI:

```
plugins/destination/plugin.go:191:11: cannot use p.setPKsForTables(tables) (value of type schemas.Schemas) as schema.Schemas value in assignment
```

Reply:
Oh also `array.DiffString` is gone I see yeah, that name just needs to change (I'll fix it)